docs/MANUAL_GCP_DEPLOYMENT.md

```diff
@@ -24,6 +24,6 @@ Client Key Pairs:
 
 5. On Cloud Run, click on the name of the application you want to mount your secrets as volume on to and click on "Edit & Deploy New Revision" on the top of the screen.
 
-![GCP Deploy & Edit](assets/gcp-edit-&-deploy.png)
+![GCP Deploy & Edit](docs/gcp-edit-&-deploy.pngg)
```

Comment:
a typo was introduced in the filename.

@redat97 let's file a task to automate testing of markdown documents links

Reply:
https://github.com/ai-cfia/devops/issues/13

Reply:
changed